secondQuestion.js

```diff
@@ -0,0 +1,18 @@
+function dot_product(v1, v2) {
```

Comment:
JS functions follow _camelCase_ naming convention not `snake_case`. Also it better to use action verbs to naming function to reflect what the function does such as `getDotProduct` or `calculateDotProduct`.